runtime/interpreter/value.go

```diff
@@ -37,6 +37,56 @@ import (
 	"github.com/onflow/cadence/runtime/trampoline"
 )
 
+func SignedBigIntToBigEndianBytes(bigInt *big.Int) []byte {
```

Comment:
Maybe move these functions further down in the file. Also, do they need to be exported?

Reply:
I intentionally exported them because I thought we'd also export them from `cadence`? If it's not needed, I'll make them private.

Also, they're used by a bunch of value types, I think I'll just move them to a separate file 👍

Reply:
Oh, that's a great point! In that case I agree that we should export them.